helm/methode-article-image-set-mapper/values.yaml

```diff
@@ -9,6 +9,8 @@ image:
   repository: coco/methode-article-image-set-mapper
   pullPolicy: IfNotPresent
 resources:
+  requests:
+    memory: 30Mi
```

Comment:
I would suggest a higher value here, somewhere around 80Mi-s for the lower limit, or even a value around 160Mi.

Reply:
Changed.